sdk/samples/iot/hub/windows/how_to_iot_hub_samples_windows.md

```diff
@@ -262,7 +261,7 @@ _The following was run on Microsoft Windows 10.0.18363.836._
     On the Azure Portal,
     - Go to your Azure IoT hub page.
     - Click on "IoT devices" under "Explorers".
-    - From the list of devices, click on your device (created on step 6).
+    - From the list of devices, click on your device (created on step 7).
```

Comment:
The step numbers were incorrect because there were duplicates, so I fixed those. @ewertons, please review to make sure they refer to the right number now.

Reply:
looks good